pubspec.yaml

```diff
@@ -16,17 +15,12 @@ publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 # https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
 # In Windows, build-name is used as the major, minor, and patch parts
 # of the product and file versions while build-number is used as the build suffix.
-version: 0.0.1+1
+version: 0.0.01+1
```

Comment:
The versioning will be done as 0.0.01+1, then 0.0.02+2, etc.
The version for every minor release, 0.0.0(x+1)+build-number and for every major release, 0.0.(x+1).0+build-number will be used.